readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -460,26 +462,36 @@ def test_make_build_commands(self):
         )
         self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
         build = resp.data
-        now = datetime.datetime.utcnow()
+        now = timezone.now()
+        start_time = now - datetime.timedelta(seconds=5)
+        end_time = now
         resp = client.post(
             '/api/v2/command/',
             {
-                'build': build['id'],
-                'command': 'echo test',
-                'description': 'foo',
-                'exit_code': 0,
-                'start_time': str(now - datetime.timedelta(seconds=5)),
-                'end_time': str(now),
+                "build": build["id"],
+                "command": "echo test",
+                "description": "foo",
+                "exit_code": 0,
+                "start_time": start_time,
+                "end_time": end_time,
             },
             format='json',
         )
         self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
         resp = client.get('/api/v2/build/%s/' % build['id'])
         self.assertEqual(resp.status_code, 200)
         build = resp.data
-        self.assertEqual(len(build['commands']), 1)
-        self.assertEqual(build['commands'][0]['run_time'], 5)
-        self.assertEqual(build['commands'][0]['description'], 'foo')
+        self.assertEqual(len(build["commands"]), 1)
+        self.assertEqual(build["commands"][0]["command"], "echo test")
+        self.assertEqual(build["commands"][0]["run_time"], 5)
+        self.assertEqual(build["commands"][0]["description"], "foo")
+        self.assertEqual(build["commands"][0]["exit_code"], 0)
+        self.assertEqual(
+            dateutil.parser.parse(build["commands"][0]["start_time"]), start_time
```

Comment:
so annoying that drf doesn't return the date in python's isoformat compatible format :x